app/controllers/agents/rdv_wizard_steps_controller.rb

```diff
@@ -10,6 +10,8 @@ class Agents::RdvWizardStepsController < AgentAuthController
   def new
     @rdv_wizard = rdv_wizard_for(query_params)
     @rdv = @rdv_wizard.rdv
+    @user = User.find(params[:user]) if params[:user]
```

Comment:
this is not necessary. 
`user_ids` is already an accepted query param, so you can use it directly without changing this controller. it will be sent to the rdv_wizard which will hydrate the Rdv with it. introducing a new `user` param will cause conflicts between the 2.